data/coalition/heliarch intro.txt

```diff
@@ -2014,10 +2014,8 @@ mission "Heliarch Drills 3"
 		government "Heliarch Test Dummy"
 		personality disables staying nemesis heroic
 		system "Torbab"
-		fleet
-			names "heliarch"
-			variant
-				"Heliarch Punisher (Scrappy)"
+		ship "Heliarch Punisher (Scrappy)" "Awaiting Adversary"
+		dialog "The Punisher's captain tries to hail you as the ship collapses around them, but you never get to hear their final cries. You have failed to simply disable the ship, and doomed the entire crew. It seems that you will not be welcome in Coalition space any longer."
```

Comment:
```suggestion
		ship "Heliarch Punisher (Scrappy)" "Awaiting Adversary"
		on kill
			dialog `The Punisher's captain tries to hail you as the ship collapses around them, but you never get to hear their final cries. You have failed to simply disable the ship, and doomed the entire crew. It seems that you will not be welcome in Coalition space any longer.`
```

Reply:
Here I'm using the `on ---` condition for the `npc` to define a dialogue to appear on the `kill` of the NPC. Otherwise, by default, a dialogue shows up upon completion of the given `npc` condition, in this case being to `disable` it.